project.godot

```diff
@@ -49,7 +49,7 @@ window/per_pixel_transparency/enabled=true
 
 [editor_plugins]
 
-enabled=PoolStringArray( "res://addons/protobuf/plugin.cfg", "res://addons/version_restrict/plugin.cfg" )
```

Comment:
Lets revert this as well.

@klingbolt can you look into this and see what we can do about either including the plugin (if the licenses are compatible) or removing it from this plugin list.